src/App.vue

```diff
@@ -95,13 +95,11 @@ export default {
     .then(response=>response.json())
     .then(json=>{
       console.log(JSON.stringify(json,null,'\t'));
-      this.taskDatas = json.map(x=>Object.assign(x,{
-        title : x.todoText,
-        regDate:x.regDate.replace(/\//g,'-')
```

Comment:
regDate format is different of html5 specification.
need to data convert on either db or js.
this code is need.